### ISSUE TITLE
ReduceLROnPlateau Doesn't Trigger on Tensorflow-gpu                1.13.1

### ISSUE THREAD

**Issue by jsl303** [tensorflow/tensorflow#28094]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Debian GNU/Linux 9.8 (stretch) (GNU/Linux 4.9.0-8-amd64 x86_64)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): conda install -c anaconda tensorflow-gpu
- TensorFlow version (use command below): 1.13.1
- Python version: 3.7.1
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.0
- GPU model and memory: Tesla P4 7611MiB
You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
I added "ReduceLROnPlateau(patience=1, monitor='val_loss', verbose=1)" as part of tf.keras.model.fit callbacks, but it never gets triggered. Other callbacks like ModelCheckpoint with the same monitor works fine.
However, the same code works on tensorflow cpu version.
I tried:
1. ReduceLROnPlateau as only callback
2. set mode to min
3. Different monitor

**Describe the expected behavior**
It should reduce lr when val_loss doesn't improve.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

from tensorflow.keras.models import Sequential
from tensorflow.keras.callbacks import ModelCheckpoint, ReduceLROnPlateau, EarlyStopping
model= Sequential()
...
earlyStopping = EarlyStopping(patience=100, monitor='val_loss', verbose=1, restore_best_weights=True)
checkpoint = ModelCheckpoint('weights/model {epoch:02d} {val_loss:.3f} {val_acc:.3f} {loss:.3f} {acc:.3f}.h5', monitor='val_loss', verbose=1, save_best_only=True)
reduce_lr_loss = ReduceLROnPlateau(patience=1, monitor='val_loss', verbose=1)
callbacks = [earlyStopping, checkpoint, reduce_lr_loss]
	model.fit(x_train, y_train, batch_size=32, epochs=1000, validation_data=(x_test, y_test), callbacks=callbacks, verbose=1)

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by muddham** [comment]
@jsl303 This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new/choose). Thanks!

**Comment by muddham** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
